eclipse-scout-core/src/testing/JasmineScoutUtil.ts

```diff
@@ -130,5 +130,90 @@ export const JasmineScoutUtil = {
         });
       }
     });
+  },
+
+  /**
+   * Asserts that every page has an uuid and a specific {@link PageParamDo}, if required.
+   */
+  assertPageCompleteness(options?: PageCompletenessOptions) {
+    options = options || {};
+    let pagesNotRequiringUuid: Set<Constructor<Page> | AbstractConstructor<Page>> = new Set([PageWithNodes, PageWithTable, AutoLeafPageWithNodes, ...options.pagesNotRequiringUuid || []]);
```

Comment:
The pattern `Constructor<Page> | AbstractConstructor<Page>` is a bit ugly. Can we simplfy this? Maybe only specify AbstractConstructor, it seems to cover "normal" constructors as well. Or rename `AbstractConstructor` to `Class`?

Reply:
As discussed: It is ugly, but we postpone this topic. We would need 3 types and decide for every usage which is the correct one.